server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -2022,7 +2022,14 @@ export class AgenticChatController implements ChatHandlers {
                     // — DEFAULT ⇒ Only MCP tools, but can also handle generic tool execution messages
                     default:
                         // Get original server and tool names from the mapping
-                        const originalNames = McpManager.instance.getOriginalToolNames(toolUse.name)
+                        let originalNames
+                        try {
+                            originalNames = McpManager.instance.getOriginalToolNames(toolUse.name)
+                        } catch (error) {
```

Comment:
we can try/catch, but we want the actual error in case it is because of other than initialization